pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -726,6 +726,15 @@ These ``DNSRule``\ s be one of the following items:
 
   :param int code: The opcode to match
 
+.. function:: PayloadSizeRule(comparison, size)
+
+  .. versionadded:: 1.9.0
+
+  Matches queries or responses whose DNS payload size is equal, greater, greater or equal, smaller or smaller or equal to the specified size.
```

Comment:
the list inside this sentence is a bit awkward

```suggestion
  Matches queries or responses whose DNS payload size fits the given comparison.
```

perhaps? the actual operator list is just below